components/esb-tools/plugins/org.wso2.integrationstudio.gmf.esb.diagram/src/org/wso2/integrationstudio/gmf/esb/diagram/custom/deserializer/DeserializerUtils.java

```diff
@@ -33,11 +33,12 @@
 import org.eclipse.gmf.runtime.notation.Node;
 import org.wso2.integrationstudio.gmf.esb.diagram.custom.AbstractConnectorEditPart;
 import org.wso2.integrationstudio.gmf.esb.diagram.custom.AbstractMediatorFlowCompartmentEditPart;
+import org.eclipse.emf.common.notify.Notifier;
```

Comment:
Shouldn't this import come before `import org.eclipse.emf.ecore.EObject;`

Reply:
Fixing : [1129](https://github.com/wso2/integration-studio/pull/1129)